src/main/java/com/amazon/opendistroforelasticsearch/knn/index/KNNMethodContext.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+/*
```

Comment:
only SPDX license is required. The amazon is not applicable anymore.

Reply:
Is this for all files? Or just new ones? Or just modified ones?

Reply:
New file only. I already added to existing file, if i missed, please add it too.